lib/cyrusdb_twoskip.c

```diff
@@ -1510,7 +1502,7 @@ static int myfetch(struct dbengine *db,
             const char **data, size_t *datalen,
             struct txn **tidptr, int fetchnext)
 {
-    int r = 0;
```

Comment:
Even though it's impossible to get to the end without initialising 'r', I'd still set it as a habit because it makes a class of mistakes less likely.